.github/workflows/job_cxx_unit_tests.yml

```
@@ -127,7 +132,13 @@ jobs:
         if: ${{ fromJSON(inputs.affected-components).PDPD_FE.test && runner.os != 'Windows' }} # Ticket: 149651
         run: |
           ${{ env.SOURCE_COMMAND }} ${{ env.SETUPVARS }}
-          ${{ env.INSTALL_TEST_DIR }}/paddle_tests --gtest_print_time=1 --gtest_output=xml:${{ env.INSTALL_TEST_DIR }}/TEST-PaddleTests.xml
+          ${{ env.INSTALL_TEST_DIR }}/paddle_tests --gtest_filter="-PaddleFuzzyOpTest*" --gtest_print_time=1 --gtest_output=xml:${{ env.INSTALL_TEST_DIR }}/TEST-PaddleTests.xml
+
+     - name: PaddlePaddle frontend Fuzzy Op tests
```

The `- name` is misaligned:
```suggestion
      - name: PaddlePaddle frontend Fuzzy Op tests
```